decidim-core/app/models/decidim/push_notification_message.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Decidim
+  # A messsage from a conversation that will be sent as a push notification
+  class PushNotificationMessage
+    class InvalidActionError < StandardError; end
+
+    def initialize(recipient:, conversation:, message:)
+      @recipient = recipient
+      @conversation = conversation
+      @message = message
+    end
+
+    attr_reader :recipient, :conversation, :message
+
+    include SanitizeHelper
+
+    def body
+      decidim_escape_translated(message)
+    end
+
+    def user
+      recipient
+    end
```

Comment:
I would use an alias method, or an endless method.

Reply:
Alias seems more clear. Done in 8aa2e7f9debe4b686f767c227cb4bff95847d9d7